src/group/service.ts

```diff
@@ -0,0 +1,70 @@
+import database from '../database';
+import { User } from '../user/types';
+import { GroupMember } from './types';
+
+type GroupMemberRecord = {
+  id: number,
+  identity: string,
+  nickname: string,
+  created: number,
+  active: number,
+  type: number
+};
+
+const fieldNames = [
+  'id',
+  'identity',
+  'nickname',
+  'created',
+  'active',
+  'type'
+];
+
+/**
+ *  Checks if the user is a type group and returns true or false
+ */
+export async function isGroup(user: User): Promise<boolean> {
+
+  const query = 'SELECT type FROM users WHERE id = ? AND type = 3';
```

Comment:
You don't need to ask the database for the type, because the `user` parameter already has a `type` property.

Reply:
embarrassing :(